mypy/tvar_scope.py

```diff
@@ -68,6 +68,7 @@ def bind(self, name: str, tvar_expr: TypeVarExpr) -> TypeVarDef:
 
     def get_binding(self, item: Union[str, SymbolTableNode]) -> Optional[TypeVarDef]:
         fullname = item.fullname if isinstance(item, SymbolTableNode) else item
+        assert fullname is not None
```

Comment:
Previously this might have been successful even if `fullname` or `item` is `None`, but apparently the original intention was that `fullname` should be defined here.